sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapWithNamedConventionsExamples.cs

```diff
@@ -94,11 +129,30 @@ private void UpsertNamedConventionsToLusid()
         }
         
         [TestCase(ModelSelection.ModelEnum.SimpleStatic)]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
         public void InterestRateSwapWithNamedConventionsValuationExample(ModelSelection.ModelEnum model)
         {
             var irs = InstrumentExamples.CreateSwapByNamedConventions();
             UpsertNamedConventionsToLusid();
             CallLusidGetValuationEndpoint(irs, model);
         }
+        
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void InterestRateSwapWithNamedConventionsInlineValuationExample(ModelSelection.ModelEnum model)
```

Comment:
VS is complaining that this method is the exact same as the one above it. Again, how is inline different?